CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Start using protocol name `halbit` instead of `halight` for Bitcoin swaps done on the Lightning network.
```

Comment:
This is not a change to the previous release right? So I think it is not worth mentioning :)

Reply:
Fair point. Thanks.